src/azul/service/responseobjects/cart_item_manager.py

```diff
@@ -23,19 +23,16 @@ def __init__(self):
         self.dynamo_accessor = DynamoDataAccessor()
         self.user_service = UserService()
 
+    @property
+    def default_cart_id_alias(self):
+        return "default"
```

Comment:
I do not like this. None is a much better default than a magic string. This just risks that defects in the code create a cart whose ID is actually `default` rather than failing early and obviously with an NPE.

Reply:
I agree with you on using `None`. I was so under the weather yesterday that I didn't get the chance to discuss with you on this issue.

Currently, as specified in the [RFC](https://github.com/DataBiosphere/azul/blob/develop/rfc/cart-api-v1.md#default-cart---todo), `default` is a magic string representing the default cart. Suppose User A has the default cart with ID `abc123` and he or she wants to get the information on the default cart, User A may send a request by sending either:

```http
GET /resources/carts/abc123
```

or

```http
GET /resources/carts/default
```

While the code in the `develop` branch did mistakenly create a default cart whose ID is `default`, the original consideration on how the service should create the default cart is pretty much the same as what `CartItemManager.create_cart` (in this PR) does. Hence, `CartItemManager.create_cart` in this PR is no longer create a cart whose ID is `default`.

Now, the question is which part of the code should handle `default` (the magic string). By not having `CartItemManager` handle the magic string, we can let request handling methods in `lambdas/service/app.py` to handle the magic string. The requesting handling code should be something like:

```python
@app.route('/resources/carts/{cart_id}', methods=['GET'], cors=True)
def get_cart(cart_id):
    user_id = get_user_id()
    manager = CartItemManager()
    try:
        cart = manager.get_cart(user_id, None if cart_id == "default" else cart_id)
    except ResourceAccessError:
        raise NotFoundError('Cart does not exist')
    else:
        return transform_cart_to_response(cart)
```